tests/test_scenarios.py

```diff
@@ -105,31 +105,32 @@ def test_pii(clean_global_registry):
             ]
 
     scenario['sites'] = [
-            Site(
-                'site1', 'party1', 'party1_ns',
-                {'id:party1_ns/dataset/pii1': 42}, scenario['rules-party1']),
-            Site(
-                'site2', 'party2', 'party2_ns',
-                {'id:party2_ns/dataset/pii2': 3}, scenario['rules-party2']),
-            Site('site3', 'ddm', 'ddm_ns', {}, scenario['rules-ddm'])]
+            Site(name='site1', administrator='party1', namespace='party1_ns',
+                 stored_data={'id:party1_ns/dataset/pii1': 42},
+                 rules=scenario['rules-party1']),
+            Site(name='site2', administrator='party2', namespace='party2_ns',
+                 stored_data={'id:party2_ns/dataset/pii2': 3},
+                 rules=scenario['rules-party2']),
+            Site(name='site3', administrator='ddm', namespace='ddm_ns',
+                 stored_data={'id:ddm_ns/software/combine': 42,
```

Comment:
I've just added random numbers here. It seems weird to have to pass an integer here, the data passed here should at some point represent the actual computation step right?

Reply:
I think it would be best to change this to Any for now, so that we can store an int for the data assets and a ComputeAsset object for the compute assets. Eventually it should become an Asset, but we cannot store data then because we don't have a DataAsset class yet. Ultimately it may even become a simple string that identifies a container image...

Reply:
I fixed this by actually implementing the DataAsset class. This simplified things a lot.